scripts/release.py

```diff
@@ -0,0 +1,199 @@
+import json
+import os
+import re
+import subprocess
+
+from dotenv import load_dotenv
+from github import Github
+
+
+"""This release notes script is built to create a release notes draft for release candidates, patches, and minor releases.
+
+Setup:
+1. Create a `.env` file in the scripts directory.
+2. Create Github token: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-personal-access-token-classic
+3. Give the Github token repo, user, audit_log, and project permissions.
+2. Add `export GH_TOKEN=<github token>` to the `.env` file.
+
+Usage:
+Required:
+    BASE - The base branch you are building your release candidate, patch, or minor release off of.
+        If this is a rc1, then just specify the branch you'll create after the release is published. e.g. BASE=1.9
+
+Optional:
+    RC - Whether or not this is a release candidate. e.g. RC=1 or RC=0
+    PATCH - Whether or not this a patch release. e.g. PATCH=1 or PATCH=0
+    
+Examples:
+Generate release notes for next release candidate version of 1.11: `BASE=1.11 RC=1 python release.py`
+
+Generate release notes for next patch version of 1.13: `BASE=1.13 PATCH=1 python release.py`
+
+Generate release notes for the 1.15 release: `BASE=1.15 python release.py`
+"""
+
+home_dir = os.path.expanduser("~")
+env_loaded = load_dotenv(dotenv_path="%s/.env" % home_dir)
+if not env_loaded:
+    raise ValueError("No envars were loaded from .env file. Please follow the instructions in the script.")
```

Comment:
We can just do a `if not os.getenv("GH_TOKEN"):` check here instead of the `.env` file as mentioned in my above comment?